models/action.go

```diff
@@ -136,6 +137,16 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```

Comment:
Thanks. Now the question is index `0` is actually the index of issue, not the title...